src/rdf4cpp/parser/ParsingState.hpp

```diff
@@ -51,10 +41,15 @@ struct ParsingState {
     /**
      * The node storage to put the parsed quads into
      */
-    node_storage_type node_storage = storage::default_node_storage;
+    storage::DynNodeStoragePtr node_storage = storage::default_node_storage;
```

Comment:
can we use the alias defined above (`node_storage_type`)?

Reply:
was planning to remove that one. Apparently forgot to do that